pkg/kubelet/userns/userns_manager.go

```diff
@@ -386,6 +386,8 @@ func (m *UsernsManager) createUserNs(pod *v1.Pod) (userNs userNamespace, err err
 func (m *UsernsManager) GetOrCreateUserNamespaceMappings(pod *v1.Pod, runtimeHandler string) (*runtimeapi.UserNamespace, error) {
 	featureEnabled := utilfeature.DefaultFeatureGate.Enabled(features.UserNamespacesSupport)
 
+	// TODO: If the default value for hostUsers ever changes, change the default value of
```

Comment:
Not sure why we need this comment.

Reply:
We are assigning a default false value to the userNamespacesEnabled based on the current default value of hostUsers. Added this comment to enable consistency.

Reply:
yeah I like this comment because for most of the host* fields we default to false, but userns is a *bool whose nil pointer is treated as true. That could change one day, and I think a note about that is relevant.